### ISSUE TITLE
My Representatives has no information 

### ISSUE THREAD

**Issue by mdwest** [CiviWiki/OpenCiviWiki#210]
The **My Representatives** feature is not currently implemented, so should not be displaying in the UI.

## Task
Remove the empty **My Representatives** section on the right side of the sign-up wizard:

![Screenshot_20190510_102715](https://user-images.githubusercontent.com/17307/57510226-e2018e80-730e-11e9-94f7-5942a644fdbe.png)

**Comment by TomaszKolek** [comment]
@mdwest can we close this one?

**Comment by brylie** [comment]
I registered and mentioned that my location was Denver, Colorado, but no data seemed to populate. On which environment should this be tested?

**Comment by brylie** [comment]
@TomaszKolek there still seems to be an issue here. As mentioned above, this step does not seem to work, perhaps it broke after the rebase:

![Screenshot_20190510_102715](https://user-images.githubusercontent.com/17307/57510014-5c7dde80-730e-11e9-897b-4a3efa236b54.png)

Please review this issue.

**Comment by TomaszKolek** [comment]
I probably misunderstood the issue when I wanted to close it. I thought that the code that shows reps in "My reps" tab is not working.

Displaying reps in my reps tab is working.

Displaying my reps in the setup page wasn't implemented and it seems its separate feature because we have not implemented sending address via API. For now, the backend will know about your location once you click Finish button not once you choose the address.

Basically, we have logic for getting reps but we have no logic to send address via API and get reps without clicking Finish button.

As a low hanging fruit, we could change the copy under my reps on the setup page? Something like you will be able to see your reps in my reps tab on the profile page.

**Comment by torabshaikh** [comment]
@brylie I think I understand the Issue. Let me know if I am wrong. the 'my Reps' should show some data from API call when user selects an address but the address is not getting submitted until user click the finish button. So we need to call the API and populate the 'my reps' tab when user selects the address. Is that right?

**Comment by brylie** [comment]
@torabshaikh in this case, we just want to **remove the My Representatives section** from the template.

**Comment by torabshaikh** [comment]
@brylie That sounds plausible. I will be fixing this then. Do I need to know anything else while setting up the environment other than the things mentioned in the [local setup](https://github.com/CiviWiki/OpenCiviWiki/blob/dev/readmes/local-setup.md) file?

**Comment by brylie** [comment]
You can try the local setup or the [docker setup](https://github.com/CiviWiki/OpenCiviWiki/blob/dev/readmes/docker-setup.md), whichever works for you.

**Comment by torabshaikh** [comment]
@brylie I have removed 'My Representative section' from the template and fixed 'locate me' button's position accordingly. Please review and merge #519 .

**Comment by torabshaikh** [comment]
@brylie Is there any other issue I can work on?

**Comment by brylie** [comment]
We have a few issues that are marked as good first issue:
https://github.com/CiviWiki/OpenCiviWiki/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
